lib/bindings/AMQPBinding.js

```diff
@@ -173,8 +180,10 @@ function start(callback) {
 
     function assertQueue(callback) {
         config.getLogger().debug(context, 'asserting queues');
-        amqpChannel.assertQueue(queue, {exclusive: false}, callback);
-        amqpChannel.assertQueue(queue + '_commands', {exclusive: false}, callback);
+        amqpChannel.assertQueue(queue, {exclusive: false}, function () {
```

Comment:
...and this other one, pls.

Reply:
All AMQP related changes are explained in https://github.com/telefonicaid/iotagent-json/pull/339#discussion_r231403078

Reply:
NTC (let's move dicussing to the cited comment)